### PR TITLE
add verbose option

### PR DESCRIPTION
**Pull request by falkodev**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -168,7 +170,8 @@ module.exports = {
             `apostrophe-i18n:${argv.locale}`,
             async () => fs.writeJson(file, translations, { spaces: 2 })
           );
-          console.timeEnd(`${argv.locale} done in`);
+
+          argv.verbose && console.timeEnd(`${argv.locale} done in`);
```
Comment (boutell) [inline index.js:174]: Please put this in an if statement for consistency with our coding style.

File: index.js [hunk 2]
```diff
@@ -245,11 +248,11 @@ module.exports = {
 
     self.on('apostrophe:modulesReady', 'generateJSONs', async function() {
       if (options.generateAtStartup) {
-        console.time('Total time');
+        options.verbose && console.time('Total time');
```
Comment (boutell) [inline index.js:251]: Please use `if`.

Comment (falkodev) [reply index.js]: Done

File: index.js (hunk 1 above)
Comment (falkodev) [reply index.js]: Done

**Comment by abea** [comment]
(also a dated changelog and version bump)

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
-## 1.0
+## 1.0.0
 
 Initial release.
+
+# 1.1.0
```
Comment (abea) [inline CHANGELOG.md:7]: Small thing here. The heading level should be `##` and the newer one should be at the top. And date it to the next scheduled release, please.

Comment (falkodev) [reply CHANGELOG.md]: Done

**Pull request merged**